angular/lib/src/lib/modal/modal.component.ts

```diff
@@ -213,25 +213,35 @@ export class ModalComponent implements OnChanges, AfterContentChecked {
 	 */
 	@Input('auClassName') className: string | undefined;
 
-	@Input('auSlotStructure') slotStructure: SlotContent<ModalContext>;
+	// eslint-disable-next-line @agnos-ui/angular-check-props
+	@Input('auSlotStructure') slotStructure: SlotContent<ModalContext<Data>>;
 	@ContentChild(ModalStructureDirective, {static: false})
-	slotStructureFromContent: ModalStructureDirective | null;
+	slotStructureFromContent: ModalStructureDirective<Data> | null;
 
-	@Input('auSlotHeader') slotHeader: SlotContent<ModalContext>;
+	// eslint-disable-next-line @agnos-ui/angular-check-props
```

Comment:
@divdavem the angular-check-props plugin was failing and am not really sure why...

Reply:
Thx !